components/ProjectCard.tsx

```diff
@@ -42,20 +44,20 @@ const ProjectCard: React.FC<ProjectCardProps> = ({
   return (
     <figure className={cardStyle}>
       <Link href={`/projects/${slug}`} passHref>
-        <div className="relative h-64 w-full">
+        <div className="flex h-36 w-full sm:h-52">
           <Image
             alt={title}
             src={coverImage}
-            layout="fill"
-            objectFit="cover"
-            objectPosition="50% 50%"
+            width={1200}
+            height={800}
+            style={{ objectFit: 'fill', ...customImageStyles }}
```

Comment:
I think `fill` is what messes up the aspect ratio, which imho should remain constant.

Resize the viewport to various sizes to see how images get squished now:
![Screenshot 2023-12-20 at 14 11 35](https://github.com/OpenSats/website/assets/109058/ae638439-bb39-41e3-ae2f-49e370d14aae)
![Screenshot 2023-12-20 at 14 11 30](https://github.com/OpenSats/website/assets/109058/a026430c-6c13-4dce-a598-7d846d7fbe1d)
![Screenshot 2023-12-20 at 14 11 15](https://github.com/OpenSats/website/assets/109058/62e41499-fca3-480c-bf36-1500c4e77bd1)

Reply:
Okay I see, makes sense. 2 of the images are horizontal and don't look great when `cover` is used. I'm checking now to see if the image is horizontal and applying `fill` to it if that's the case. Looks better overall now IMO. Let me know if you've got a better idea.